### DISCUSSION TITLE
Live Chat Filter

### DISCUSSION THREAD

**Discussion by Syzygos** [ReVanced/revanced-discussions#129] [Question]
A way to filter out live chat messages during live streams. Similar to [LiveTL](https://livetl.app/) or [ ytcFilter](https://github.com/RomainLK/ytc-filter).

The most common use for this is to grab messages made by live translators in live chats and display them on their own so viewers that don't understand the language can follow along with what the streamer is saying and I think it's a very useful feature to have.

I'm not entirely sure this would even be possible, but it doesn't hurt to ask!

**Comment by oSumAtrIX** [comment]
Something like this should be done via a service. So you stream to that service and the service streams to YouTube. The service also adds whatever you need to the stream. Having it in ReVanced? Idk..

**Comment by gyroninja** [comment]
In case you aren't familiar with what the OP was referring to, these translations are being provided by a third party who often is not working together with the streamer. There are just "random" people in chat who are helping other understand what is going on by translating the stream. In a very fast chat, in one that primarily speaks a language you don't, or if you just don't like chat messages there is a benefit to filtering out messages other than these translations. This is a niche feature, but I just wanted to explain what it is.

**Comment by psy0ch** [comment]
A browser plugin that works on the idea that gyroninja mentioned is "LiveTL" btw http://livetl.app/
Niche feature indeed, but neat for people that are following international content creators

**Comment by TheJeterLP** [comment]
Probably out of scope for ReVanced.
